README.md

```diff
@@ -1,18 +1,18 @@
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/1e172cdf699645b59567032dd1ae5cab)](https://www.codacy.com/manual/Angourisoft/MathS?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Angourisoft/MathS&amp;utm_campaign=Badge_Grade)
```

Comment:
Yeah, Codacy definitely is too restrictive over style choices.

Reply:
Rather stupid and glitchy, to be honest.